3338-1_1.xml

```diff
@@ -0,0 +1,123 @@
+-->
+
+<LWM2M  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://openmobilealliance.org/tech/profiles/LWM2M.xsd">
+	<Object ObjectType="MODefinition">
+		<Name>Buzzer</Name>
+		<Description1>This IPSO object should be used to actuate an audible alarm such as a buzzer, beeper, or vibration alarm. There is a dimmer control for setting the relative loudness of the alarm, and an optional duration control to limit the length of time the alarm sounds when turned on. Each time "true" is written to the On/Off resource, the alarm will sound again for the configured duration. If no duration is programmed or the setting is "false", writing a "true" to the On/Off resource will result in the alarm sounding continuously until a "false" is written to the On/Off resource.
+        </Description1>
+		<ObjectID>3338</ObjectID>
+		<ObjectURN>urn:oma:lwm2m:ext:3338:1.1</ObjectURN>
+		<LWM2MVersion>1.0</LWM2MVersion>
+		<ObjectVersion>1.1</ObjectVersion>
+		<MultipleInstances>Multiple</MultipleInstances>
+		<Mandatory>Optional</Mandatory>
+		<Resources>
+			<Item ID="5850">
+				<Name>On/Off</Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Mandatory</Mandatory>
+				<Type>Boolean</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>On/off control. Boolean value where True is On and False is Off.</Description>
+			</Item>
+			<Item ID="5548">
+				<Name>Level</Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration>0..100</RangeEnumeration>
+				<Units>/100</Units>
+				<Description>Audio volume control, float value between 0 and 100 as a percentage.</Description>
+			</Item>
+			<Item ID="5521">
+				<Name>Delay Duration</Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>s</Units>
+				<Description>The duration of the time delay.</Description>
+			</Item>
+			<Item ID="5525">
+				<Name>Minimum Off-time</Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Mandatory</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>s</Units>
+				<Description>The off time when On/Off control remains on.</Description>
+			</Item>
+			<Item ID="5750">
+				<Name>Application Type</Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>String</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>The application type of the sensor or actuator as a string depending on the use case.</Description>
+			</Item>
+			<Item ID="5518">
+				<Name>Timestamp</Name>
```

Comment:
Is timestamp in buzzer intentional? "when measurement was performed" seems a bit odd here.

Reply:
@akeranen your comments are in a branch that was previously agreed by IPSO WG. 
The plan was to update this branch with the same changes as in Common. At that time there will be an option to update this content.

Reply:
Sounds good. Where should we make a note of that so that it's not forgotten?

Reply:
@akeranen @dirtyha    At the last IPSO call, it was agreed:

Before asking Harri  (Vaisala) to run his tool again to create versions 1.1 of IPSO objects Matt will provide a proposal on how to release Objects in this registry: block or single releases.
Once this is clear the staff will prepare the repository for that particular of release type. Once the repository is ready, we can ask to Harri to run his tool and create a new set of IPSO Objects. If we try to do it know we will have near twice the amount work when preparing the release of objects in this repository.

This release that I am referring to is that currently, all the versions of an object is in the same branch. But the group wants to remove versions from the filename, this implies that there will be a single version of an object in the same branch. Previous versions will be stored in  the "release branches".